src/core/github.js

```diff
@@ -153,6 +153,9 @@ export async function run(conf) {
     repoURL: ghURL.href,
     apiBase: githubAPI,
     fullName: `${org}/${repo}`,
+    issuesURL: issueBase,
+    pullsURL: newProps.pullBase,
+    newIssuesURL: new URL("./new", issueBase).href,
```

Comment:
This will show the issue template chooser if a template exists, otherwise redirect to `/issues/new`:
```suggestion
    newIssuesURL: new URL("./new/choose", issueBase).href,
```